src/bundle/Resources/public/scss/_custom.tooltip.scss

```diff
@@ -62,6 +62,54 @@
         box-shadow: 0 calculateRem(4px) calculateRem(6px) rgba(135, 135, 135, 0.35);
     }
 
+    &--dark {
+        .ibexa-tooltip {
+            &__inner {
+                color: $ibexa-color-white;
+                background-color: $ibexa-color-dark;
```

Comment:
I won't be cruel to you as there are other modifiers already, but I think we should consider using either mixin or (perhaps better) css variables for these colors so that you could do just
```--dark {
  --ibexa-tooltip-bg-color: $ibexa-color-dark;
}
to add new type of tooltip